genai-perf/genai_perf/parser.py

```diff
@@ -426,6 +426,12 @@ def _print_warnings(args: argparse.Namespace) -> None:
             "Custom tokenizer code can be executed. "
             "This should only be used with repositories you trust."
         )
+    if args.prompt_source == ic.PromptSource.PAYLOAD and args.output_tokens_mean:
```

Comment:
Won't this warning always go off, since we set a default mean? This will add noise. Not a blocker, but we should consider removing it in the near future. We can assume the user will do the right thing.

If users start asking for a warning, the we can always add it only in the case where a payload has an OSL provided and the user specifies a global OSL.

Reply:
Yeah. If you wanted to hide this for most folks in the short-term, you could do something like `... and args.output_token_mean != ic.DEFAULT_OUTPUT_TOKENS_MEAN`.